build.gradle

```diff
@@ -8,7 +8,7 @@ idea.project.languageLevel = '1.8'
 
 subprojects {
 
-    version='1.0.2-SNAPSHOT'
+    version='1.0.3-SNAPSHOT'
```

Comment:
Please provide a separate commit for the Version 1.0.2, so we can tag and release it.